ctf/check.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
It's fine as along as rubenwardy doesn't object to it. I got no say in it ether way.

As for why, it's simply because of maintainability.

There are test suites which support searching for tests through recursive like [busted](http://olivinelabs.com/busted/) which does isolate tests.